src/components/FAIcon.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <FontAwesomeIcon :icon="iconWithStyle" />
```

Comment:
Should we use aria-hidden on these SVGs? or is it OK not to?

Edit: the docs pointed me that it's automatic! https://fontawesome.com/v5.15/how-to-use/on-the-web/other-topics/accessibility#svg-decorative So nothing to change here

Reply:
Not necessary, it's done automatically yes